dukan/src/main/kotlin/net/thechance/dukan/api/dto/CategoryDto.kt

```diff
@@ -0,0 +1,8 @@
+package net.thechance.dukan.api.dto
+
+data class CategoryDto(
+    val id: String,
+    val icon: String,
+    val arabicTitle: String,
+    val englishTitle: String,
```

Comment:
We can return title to the client and based on language header we return either english or arabic what do you think?

```suggestion
    val title: String,
```

Reply:
make sense to me